dbus_service.py

```diff
@@ -605,22 +582,18 @@ def get_values_for_inverter(self):
         elif self.dtuvariant == constants.DTUVARIANT_OPENDTU:
             root_meter_data = meter_data["inverters"][self.pvinverternumber]
             producing = is_true(root_meter_data["producing"])
-            power = (
-                root_meter_data["AC"]["0"]["Power"]["v"]
-                if producing
-                else 0
-            )
```

Comment:
same. @henne49  I would like to to know you your thoughts about this style

Reply:
Yes, this is a matter of taste. Me, I don't like having to many lines where 2 of 5 lines are only containing 1 parenthesis.